src/sinks/util/batch.rs

```diff
@@ -101,6 +101,10 @@ where
         Self::build(inner, batch, min_size, min_size, max_linger)
     }
 
+    pub fn from_settings(inner: S, batch: B, settings: BatchSettings) -> Self {
+        Self::new_min(inner, batch, settings.size, settings.timeout.into())
```

Comment:
Do we want a minimum size to be the default? This is a little confusing, unless `BatchSettings` somehow implies minimums.

Reply:
We use `batch_with_min` everywhere so I figured this was the norm.